src/components/header-component/header-component.tsx

```diff
@@ -1,17 +1,17 @@
 import { Header } from 'components/header/header';
 import { MovieDescription } from 'components/movie-description/movie-description';
 import { MovieSearch } from 'components/movie-search/movie-search';
-import { useMovieContext } from 'contexts/movie-context';
+import { useAppSelector } from 'redux/hooks';
 
 import styles from './header-component.module.scss';
 
 export const HeaderComponent = () => {
-  const { selectedMovie, setSelectedMovie } = useMovieContext();
+  const selectedMovie = useAppSelector((state) => state.movies.selectedMovie);
```

Comment:
I strongly advise to create a separate folder called `selectors` and store all entity selectors here to make your selectors reusable across app. It really simplifies work when you decide to refactor Redux state. Take a look at my example [here](https://github.com/dotnet-fizzyy/Diploma/tree/V2/web/src/redux/selectors)